backend/python/app/graphql/resolvers/entity_resolver.py

```diff
@@ -0,0 +1,21 @@
+from ..types.entity_type import Entity, EntityInput, entity_input_to_dto_and_model
+from ..service import services
+
+import graphene
+
+class CreateEntity(graphene.Mutation):
+    class Arguments:
+        entity_data = EntityInput(required=True)
+        
+    ok = graphene.Boolean()
+    entity = graphene.Field(lambda: Entity)
+
+    def mutate(root, info, entity_data=None):
+        entity_dto, entity_model = entity_input_to_dto_and_model(entity_data)
```

Comment:
Are both the DTO and model necessary here? I think ideally the model should be hidden from the resolver and the resolver should deal with DTOs only, otherwise this defeats the purpose of having the service layer.

Disclaimer that I have not worked with GraphQL extensively but I think it would be beneficial to decouple the model (and by extension the ORM) from the GraphQL resolver.

Would line 17 work if you pass in the result of line 15?
```python
# I think it returns a dict right now but it essentially functions as an "EntityResponseDTO"
services["entity"].create_entity(entity_dto)
```